docs/manpages/zone2ldap.1.md

```diff
@@ -20,6 +22,15 @@ inserted to an LDAP server.
 --basedn=*DN*
 :    Base DN to store objects below
 
+--create-ent
```

Comment:
maybe this should be `--dnssec` if it's only used for dnssec-enabled zones? (but is this part actually true?)